parquet/src/util/bit_util.rs

```diff
@@ -672,8 +632,10 @@ impl BitReader {
         })
     }
 
-    fn reload_buffer_values(&mut self) {
-        let bytes_to_read = cmp::min(self.total_bytes - self.byte_offset, 8);
+    /// Populates `self.buffered_values`
+    #[inline]
+    fn load_buffer(&mut self) {
```

Comment:
Given there is a field called `buffer` and this function is not loading it, but rather reading *from* it and loading into `buffered_values` I recommend calling this function something slightly verbose:

```suggestion
    /// Loads up to the the next 8 bytes from `self.buffer` at `self.byte_offfset` 
    /// into `self.buffered_values`. Reads fewer than 8 bytes if there are fewer than 8 bytes left
    fn load_buffered_values(&mut self) {
```